test/runtests.jl

```diff
@@ -2,6 +2,10 @@ using PreCICE
 using Test
 
 
+libprecicePath = split(readlines(`whereis libprecice`)[], ' ')[2]
+
+PreCICE.setPathToLibprecice(string(libprecicePath))
```

Comment:
Why is this necessary at the start of the tests?

Reply:
The path to the libprecice library is set here, to be independant of the preCICE installation location in case one who runs the tests has installed it to a different location.